draft-cutler-httpbis-cookies-having-independent-partitioned-state.md

```diff
@@ -159,7 +159,9 @@ The following could be added as a new subsection of section 6.1 (Limits):
 {:quote}
 > User agents SHOULD enforce a separate per-domain limit for partitioned cookies for a particular cookie-partition-key.
 > This limit should be lower than the per-domain limit for cookies without the Partitioned attribute to prevent cookies set on different top-level sites from reaching implementation memory limits.
```

Comment:
Be consistent about SHOULD vs should. I think RFCs generally prefer all-caps even though web specs prefer lower case.

Reply:
Done.